src/applications/disability-benefits/new-686/config/chapters/chapter-2/1-veteran-formation/veteran-information.js

```diff
@@ -0,0 +1,83 @@
+import {
+  genericTextinput,
+  genericNumberInput,
+  genericDateInput,
+} from '../../../generic-schema';
+import { suffixes } from '../../../constants';
+import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import _ from 'lodash/fp';
+
+export const schema = {
+  type: 'object',
+  required: ['first', 'last', 'ssn', 'birthDate'],
+  properties: {
+    first: genericTextinput,
+    middle: genericTextinput,
+    last: genericTextinput,
+    suffix: {
+      type: 'string',
+      enum: suffixes,
+    },
+    ssn: genericNumberInput,
+    vaFileNumber: genericNumberInput,
+    serviceNumber: genericNumberInput,
+    birthDate: genericDateInput,
+  },
+};
+
+export const uiSchema = {
+  first: {
+    'ui:title': 'Your first Name',
+  },
+  middle: {
+    'ui:title': 'Your middle Name',
+  },
+  last: {
+    'ui:title': 'Your last Name',
+  },
+  suffix: {
+    'ui:options': {
+      widgetClassNames: 'usa-input-medium',
+    },
+  },
+  ssn: _.merge(_.unset('ui:title', ssnUI), {
+    'ui:title': 'Your Social Security number',
+  }),
+  vaFileNumber: {
+    'ui:title': 'Your VA file number',
+    'ui:errorMessages': {
+      pattern: 'Please enter only numbers',
```

Comment:
This could probably be the same as the message found in the burials formConfig - `Your VA file number must be 8 or 9 digits`

Reply:
I made this match the latest version of the mockup from Jim instead